notebooks/community/managed_notebooks/Recommender Systems/Recommender-system_notebook.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    query = """WITH user_prod_table AS (
</pre><p>The <strong>bigquery-public-data </strong>project is the place for public BigQuery data. The instructions to add data are at <a href="https://github.com/GoogleCloudPlatform/public-datasets-pipelines" target="_blank">GoogleCloudPlatform/public-datasets-pipelines: Cloud-native, data onboarding architecture for the Google Cloud Public Datasets program</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>